R/get_panel.R

```diff
@@ -4,16 +4,53 @@
 #' one panel (i.e., plots that are not faceted).
 #'
 #' @param plot A ggplot or gtable from which to retrieve the panel
+#' @param panel An integer indicating which panel to pull
+#' @param pattern the name of the component
```

Comment:
This doesn't seem to exist in the function.

Reply:
For this, it's because `get_panel()` and `get_panel_component()` are in the same documentation. I left it that way but can separate if you prefer.

Reply:
Ah, I see. Either way is fine, but if they're combined into one then the first line should maybe say "Retrieve the panel or part of a panel of a plot"